.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "Tests/xctest-watchOS"]
+	path = Tests/XCTest-watchOS
+	url = ../../dfed/XCTest-watchOS.git
```

Comment:
Should this be checked in?

Reply:
Having it checked in seems like the easiest thing. Note that it should be checked in under the `Tests/XCTest-watchOS` directory.

Reply:
I guess I'm actually just unclear of what this submodule is.

Reply:
This submodule is the code that does the `XCTest` on `watchOS`. It originally lived in this PR, but I broke it out into it's own repo/submodule since it isn't tightly coupled to `Valet`. Code is here: https://github.com/dfed/XCTest-watchOS

Reply:
Ah, gotcha. Is this a local url? Or can you define relative remote urls like this? Assuming it _is_ remote, can we update this to point at the absolute url for the sake of clarity?